lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -325,33 +348,38 @@ class RadarDataSet with EquatableMixin {
   RadarDataSet copyWith({
     List<RadarEntry>? dataEntries,
     Color? fillColor,
+    Gradient? gradient,
     Color? borderColor,
     double? borderWidth,
     double? entryRadius,
   }) =>
       RadarDataSet(
         dataEntries: dataEntries ?? this.dataEntries,
         fillColor: fillColor ?? this.fillColor,
+        gradient: gradient,
         borderColor: borderColor ?? this.borderColor,
         borderWidth: borderWidth ?? this.borderWidth,
         entryRadius: entryRadius ?? this.entryRadius,
       );
 
   /// Lerps a [RadarDataSet] based on [t] value, check [Tween.lerp].
-  static RadarDataSet lerp(RadarDataSet a, RadarDataSet b, double t) =>
-      RadarDataSet(
-        dataEntries: lerpRadarEntryList(a.dataEntries, b.dataEntries, t),
-        fillColor: Color.lerp(a.fillColor, b.fillColor, t),
-        borderColor: Color.lerp(a.borderColor, b.borderColor, t),
-        borderWidth: lerpDouble(a.borderWidth, b.borderWidth, t),
-        entryRadius: lerpDouble(a.entryRadius, b.entryRadius, t),
-      );
+  static RadarDataSet lerp(RadarDataSet a, RadarDataSet b, double t) {
+    return RadarDataSet(
```

Comment:
Why did you change it from a single line return to a body return style?
I asked because now we have a change for all the lines